source/adapters/hip/usm.cpp

```diff
@@ -132,7 +132,6 @@ ur_result_t USMHostAllocImpl(void **ResultPtr, ur_context_handle_t Context,
                              ur_usm_host_mem_flags_t *, size_t Size,
                              [[maybe_unused]] uint32_t Alignment) {
   try {
-    ScopedContext Active(Context->getDevice());
```

Comment:
This changes has caused `hContext` to become unused. This warnign becomes an error when `UR_DEVELOPER_MODE=ON`.